templates/wordpress/README.md

```
@@ -1,38 +1,47 @@
 
+**Updated (REST API enabled):**
+```
+"ALLOWED_METHODS": "GET|POST|HEAD|OPTIONS|PUT|DELETE"
+```
+
+## Prerequisites
 - A running WordPress upstream accessible on your network (for example a container named `mywp`).
 - The BunkerWeb UI or the ability to edit multisite settings directly.
 - Domain name(s) that will serve the WordPress instance.
 
 ## Files
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)